src/apm.js

```diff
@@ -106,6 +106,7 @@ export function getPerformanceData() {
         paint: getPerformanceEntriesByType('paint'),
         resource: getPerformanceEntriesByType('resource'),
     }
+    window?.performance?.clearResourceTimings()
```

Comment:
There's a chance this will crash for everyone using posthog-js on the server side with this config flag on. Even though babel will probably escape `?.` to this, it's best to explicitly check if window exists:

```ts
if (typeof window !== 'undefined') {
   /* code here */
}
```